io.openems.edge.batteryinverter.api/src/io/openems/edge/batteryinverter/api/ManagedSymmetricBatteryInverter.java

```diff
@@ -0,0 +1,86 @@
+package io.openems.edge.batteryinverter.api;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.exceptions.OpenemsError.OpenemsNamedException;
+import io.openems.common.exceptions.OpenemsException;
+import io.openems.edge.battery.api.Battery;
+import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.modbusslave.ModbusSlaveNatureTable;
+import io.openems.edge.common.startstop.StartStoppable;
+
+/**
+ * Represents a Symmetric Battery-Inverter that can be controlled.
+ * 
+ * <p>
+ * To indicate, that the Battery-Inverter is ready for charging/discharging, the
+ * following Channels need to be set:
+ * 
+ * <ul>
+ * <li>StartStoppable.ChannelId.START_STOP must be set to 'START'
+ * <li>No 'Fault'-StateChannels are set (i.e. 'OpenemsComponent.ChannelId.STATE'
+ * is < 3)
+ * </ul>
+ */
+@ProviderType
+public interface ManagedSymmetricBatteryInverter extends SymmetricBatteryInverter, StartStoppable {
+
+	public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
+		;
+
+		private final Doc doc;
+
+		private ChannelId(Doc doc) {
+			this.doc = doc;
+		}
+
+		public Doc doc() {
+			return this.doc;
+		}
+	}
+
+	public static ModbusSlaveNatureTable getModbusSlaveNatureTable(AccessMode accessMode) {
+		return ModbusSlaveNatureTable.of(ManagedSymmetricBatteryInverter.class, accessMode, 100) //
+				.build();
+	}
+
+	/**
+	 * Apply {@link Battery} and target Active and Reactive Power.
+	 * 
+	 * <p>
+	 * 
+	 * This is called on ManagedSymmetrcEss::applyPower()
+	 * 
+	 * @param battery          the {@link Battery}
+	 * @param setActivePower   the active power setpoint
+	 * @param setReactivePower the reactive power setpoint
+	 * @throws OpenemsNamedException on error
+	 */
+	public void apply(Battery battery, int setActivePower, int setReactivePower) throws OpenemsNamedException;
```

Comment:
As described personally, the main idea of the `apply()` method was to create one single entry-point for a Battery-Inverter (similar to `run()` for a Controller), to avoid having to handle separate different Events and so on. 

To do this, I also need the Battery, because the Battery-Inverter needs to read certain informations from it. 

As discussed, if I wanted to avoid supplying the Battery (and I had tried that), I would end up rebuilding a POJO with the same fields like the Battery.